src/app/ory/getIdentity.ts

```diff
@@ -0,0 +1,29 @@
+import { identityApi } from "@/app/ory/backendApi"
+import { Identity } from "@ory/client"
+
+export const getIdentityById = async (
+  id?: string | null,
+): Promise<Identity | undefined> => {
+  if (id === undefined || id === null) {
+    return undefined
+  }
+
+  try {
+    return (
+      await identityApi.getIdentity({
+        id,
+      })
+    ).data
+  } catch (e) {
+    return undefined
+  }
+}
+
+export const getSingleVerifiedEmailAddress = (
+  identity: Identity,
+): string | undefined => {
+  const verifiedAddresses = identity.verifiable_addresses
+  return verifiedAddresses?.length === 1 && verifiedAddresses[0].verified
```

Comment:
Maybe something like this? I think we only have one email field atm, but kratos does allow you to have more than 1 email account that can be verified. 

```suggestion
  return verifiedAddresses.filter((v) => v.verified)?.[0] || undefined
```

Reply:
I like the idea of filtering by verified, but I think it might be a better idea to fail somehow when there's an ambiguity rather than just force one email and perhaps not let the user realise that something's up. Especially since there's only one field anyway and that probably won't change until November. Do you think that would be ok, @Benehiko?

Reply:
yes that sounds good to me